app/controllers/preferences_interview_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class PreferencesInterviewController < StandardStepsController
+  def self.step_class
+    PreferencesInterview
+  end
```

Comment:
I could be wrong but I think it'll infer this without needing to explicitly define it?